openapi-cli/src/main/java/io/ballerina/openapi/generators/client/FunctionSignatureGenerator.java

```diff
@@ -473,10 +509,30 @@ private Node getHeaderParameter(Parameter parameter, NodeList<AnnotationNode> pa
                     requestBodyDoc.add(paramAPIDoc);
                 }
                 parameterList.add(payload);
+                parameterList.add(createToken((COMMA_TOKEN)));
+            }
+
+            if (next.getKey().equals(javax.ws.rs.core.MediaType.MULTIPART_FORM_DATA)
+                    && next.getValue().getEncoding() != null) {
+                List<String> headerList = new ArrayList<>();
+                for (Map.Entry<String, Encoding> entry : next.getValue().getEncoding().entrySet()) {
+                    for (Map.Entry<String, Header> header : entry.getValue().getHeaders().entrySet()) {
+                        if (!headerList.contains(header.getKey())) {
+                            Node paramh = getHeaderEncoding(header, requestBodyDoc);
```

Comment:
Shall we use a proper name instead of `paramh`

Reply:
yes. Resolved in https://github.com/ballerina-platform/openapi-tools/commit/bfcb193a81bf7f406a52321d11e64ad7920ca61f